src/js/demo-theme.js

```diff
@@ -1,28 +1,29 @@
 /**
- * demo-theme is specifically loaded right after the body and not deferred
- * to ensure we switch to the chosen dark/light theme as fast as possible.
- * This will prevent any flashes of the light theme (default) before switching.
+ * 
+ * This will allow you to load dark/light theme wihtout reloading the page
+ * and without depending on the link variables 
+ * default theme is light and can be default dark by changing the specified code below
```

Comment:
Your new comment is not a substitute for the original.

The reason the original comment is there is to explain why the script is loaded at the top of the body and not what it does. As this reason still applies please leave it intact.